include/OpenSpaceToolkit/Astrodynamics/NumericalSolver.hpp

```diff
@@ -34,7 +34,8 @@ class NumericalSolver
     enum class StepperType
     {
```

Comment:
RK4 at the top to go from "simplest" to "more complex" (nit). Adapt in bindings if agree